app/utils/detectors.py

```diff
@@ -0,0 +1,130 @@
+import re
+from typing import List, Optional
+
+import pandas as pd
+import pyarrow as pa
+from fastapi import HTTPException
+
+ID_KEY = "id"
+VALUE_KEY = "value"
+DATE_KEYS = ["date", "day", "time", "timestamp", "datetime", "dttm"]
+
+
+def _norm(name: str) -> List[str]:
+    name = str(name).lower().strip()
```

Comment:
issue: why do you need to explicitly cast name to str? is it possible that the typehint should be `str | int` or anything else?